appGeo/src/app/game/game.component.ts

```diff
@@ -1,13 +1,14 @@
 import { Component, OnInit } from '@angular/core';
-import {Player, Killer, Civilian} from '../player/player.component';
+import {Player, Killer, Civilian} from '../player/player_class_declaration';
 import{CampusMap} from '../map/campus-map.component'
 import {Chat, Message} from '../chat/chat_class_declaration'
 //A CampusMap is a Map of the Bubbles that exist on campus
 
 const INACTIVE = 0
 const ACTIVE = 1
-const CIVILIAN = 0
-const KILLER = 1
+const CIVILIAN = 7
+const KILLER = 8
```

Comment:
I'm confused by this numbering system for these. Given these are a boolean pair of constants, I don't understand why they're being switched to 7 and 8.

Also, we do have several constants like these we use in multiple files—we should probably move these all to a base file to import from.

Reply:
I did that for clarity, I'm not particularly a fan of having different constants to have the same value (could result in some tests passing when they are different things). Strictly speaking, yes, they are both boolean pair of constants. But fundamentally calling a game Inactive or Active, is different then if you are taking about Civilians or Killers.